SI/Sem.02/Pract.02/task04.cpp

```diff
@@ -0,0 +1,46 @@
+// task4.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+//�� �� ������ ��������, ����� ���� �� ��������� 1 ���� ����� - �������� �� �����, � ��������� �������� �� �������� ����� : >= 90 % : 6 +
+//>= 80 % : 6
+//>= 70 % : 5
+//>= 60 % : 4
+//>= 40 % : 3
+//< 40 % : 2
+{
+	int result;
+	cout << "Exam result:";
+	cin >> result;
+	if (result >= 90)
+	{
+		cout << " The mark is: 6+";
+	}
+	else if (result >= 80)
+	{
+		cout << "The mark is:6";
+	}
+	else if(result>=70)
```

Comment:
keep consistency and good practices.
Tabulations/Spaces between the operations and variables.
E.g: else if(result >= 70)